operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go

```diff
@@ -201,7 +201,12 @@ func (r *ObservabilityAddonReconciler) Reconcile(ctx context.Context, req ctrl.R
 				log.Error(err, "OCP prometheus service does not exist")
 				// ACM 8509: Special case for hub/local cluster metrics collection
 				// We do not report status for hub endpoint operator
-				util.ReportStatus(ctx, r.Client, obsAddon, "NotSupported", !isHubMetricsCollector)
+				if !isHubMetricsCollector {
```

Comment:
We don't have an ObservabilityAddon in the local cluster?

Reply:
Good question. The local cluster is not working as an addon, it's probably because of that.
In any case, I haven't changed the existing logic.

Reply:
Oh, missed it passed through the `isHubMetricsCollector` as args. Thanks